### PR TITLE
BFB-102: Limit the username

### PR DESCRIPTION
**Pull request by m4rtint**
**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-102

**Description**
Limits the username to be only alpha numeric.
This regex should now allow alpha number + space

**Screenshots / GIFs**
Capture

### CODE REVIEW

File: Assets/_BForBoss/_Leaderboard/Scripts/Username/InputUsername.cs [hunk 1]
```diff
@@ -1,3 +1,6 @@
+using System;
+using System.Linq;
```
Comment (unoctium1) [inline Assets/_BForBoss/_Leaderboard/Scripts/Username/InputUsername.cs:2]: I think this is an unneeded using

**Review by unoctium1 — CHANGES_REQUESTED** [review]
Mostly good, but I think its a good idea to clear out the unnecessary linq using before merging this

File: Assets/_BForBoss/_Leaderboard/Scripts/Username/InputUsername.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_BForBoss/_Leaderboard/Scripts/Username/InputUsername.cs]: Fixed

**Review by unoctium1 — APPROVED** [review]
lgtm

**Pull request merged**